modules/wizard/wizard.php

```diff
@@ -86,9 +88,9 @@ public function __construct( &$polylang ) {
 	 * @return void
 	 */
 	public static function start_wizard( $network_wide ) {
-		$options = get_option( 'polylang' );
+		$options = (array) get_option( Options::OPTION_NAME, array() );
```

Comment:
Could we use `PLL()->options` here? Any advice @Chouby?

Reply:
I don't think we can, this method is called that way:
```php
register_activation_hook( POLYLANG_BASENAME, array( 'PLL_Wizard', 'start_wizard' ) );
```

`$GLOBALS['polylang']` will be created at `'plugins_loaded'`, so it is not available at the activation hook.

Reply:
I see, thanks for the explanation 👍